src/index.js

```diff
@@ -47,6 +47,11 @@ const parserClassCache: { [key: string]: Class<Parser> } = {};
 
 /** Get a Parser class with plugins applied. */
 function getParserClass(pluginsFromOptions: $ReadOnlyArray<string>): Class<Parser> {
+
+  if (pluginsFromOptions.indexOf("decorators") >= 0 && pluginsFromOptions.indexOf("decorators2") >= 0) {
```

Comment:
I'd say to use `Array.prototype.includes` instead...... but that doesn't work on Node 4 :cry: